modules/ROOT/pages/getting-started.adoc

```diff
@@ -85,4 +85,4 @@ Follow the xref:bare-metal.adoc[Bare Metal Installation Instructions] to install
 
 == Where to report bugs and ask questions
 
-Report bugs to the https://github.com/coreos/fedora-coreos-tracker[Fedora CoreOS Tracker] and ask questions on the `#fedora-coreos` IRC channel on freenode or on the https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/[Fedora CoreOS mailing list].
+Report bugs to the https://github.com/coreos/fedora-coreos-tracker[Fedora CoreOS Tracker] and ask questions on the `#fedora-coreos` IRC channel on freenode or on the https://discussion.fedoraproject.org/c/server/coreos/[Fedora CoreOS forum] or https://lists.fedoraproject.org/archives/list/coreos@lists.fedoraproject.org/[mailing list].
```

Comment:
seems like we should use a comma and only one or?

*and ask questions on the `#fedora-coreos` IRC channel on freenode, the Fedora CoreOS forum, or on the Fedora CoreOS mailing list.*

PS my grammar skills aren't great so there might be something wrong with my suggestion